git/essentials/features-ii/diff-images-using-exiftool.md

```diff
@@ -11,6 +11,11 @@ type: normal
 
 category: feature
 
+aspects:
+  
+  - introduction
```

Comment:
I would remove `introduction`, since this requires knowledge of how `.gitattributes` work and also how the git configuration system works, this may be more appropriately tagged `workout` and `deep`.